src/Sarif/ConsoleLogger.cs

```diff
@@ -263,12 +264,12 @@ private string FormatNotificationMessage(Notification notification)
 
             if (!string.IsNullOrEmpty(notification.Id))
             {
-                sb.Append($" {notification.Id}: ");
+                sb.AppendFormat(CultureInfo.CurrentCulture, "$" +notification.Id+": ");
             }
 
             if (!string.IsNullOrEmpty(notification.RuleId))
             {
-                sb.Append($"{notification.RuleId}: ");
+                sb.AppendFormat(CultureInfo.CurrentCulture, "$" + notification.RuleId + ": ");
```

Comment:
Sorry, I missed this during my review. Michael is right, the "`$`" must be removed. To see what is was there for, do a web search for "c# interpolated strings".

In any case, you don't need to use `AppendFormat` at all. Just call


```
sb.Append(notification.RuleId);
```

... which avoids the whole question of which culture to use.